python/ray/data/preprocessors/scaler.py

```diff
@@ -141,17 +141,56 @@ def __repr__(self):
 
 
 class RobustScaler(Preprocessor):
-    """Scale values within columns based on their quantile range.
+    r"""Scale and translate each column using quantiles.
 
-    For each column, each value will be transformed to
-    ``(value - median) / (high_quantile - low_quantile)``,
-    where ``median`` , ``high_quantile``, and ``low_quantile``
-    are calculated from the fitted dataset.
+    The general formula is given by
+
+    .. math::
+        x' = \frac{x - \mu_{1/2}}{\mu_h - \mu_l}
+
+    where :math:`x` is the column, :math:`x'` is the transformed column,
+    :math:`\mu_{1/2}` is the column median. :math:`\mu_{h}` and :math:`\mu_{l}` are the
+    high and low quantiles, respectively. By default, :math:`\mu_{h}` is the third
+    quartile and :math:`\mu_{l}` is the first quartile.
+
+    .. note::
+        This scaler works well when your data contains many outliers.
 
     Args:
-        columns: The columns that will be scaled individually.
-        quantile_range: A tuple that defines the lower and upper quantile to scale to.
-                        Defaults to the 1st and 3rd quartiles: (0.25, 0.75).
+        columns: The columns to separately scale.
+        quantile_range: A tuple that defines the lower and upper quantiles. Values
+            must be between 0 and 1. Defaults to the 1st and 3rd quartiles:
```

Comment:
nit: Consistency with above.

```suggestion
            must be between 0 and 1. Defaults to the first and third quartiles:
```